### ISSUE TITLE
Problem serialize dashboard object

### ISSUE THREAD

**Issue by juanCarlos123** [FST2015PM/Plataforma#14]
Error:
catalina.core.StandardWrapperValve invoke
SEVERE: Servlet.service() for servlet [Jersey REST Service] in context with path [] threw exception [java.lang.RuntimeException: json can't serialize type : class org.json.JSONObject] with root cause
java.lang.RuntimeException: json can't serialize type : class org.json.JSONObject
	at com.mongodb.util.ClassMapBasedObjectSerializer.serialize(ClassMapBasedObjectSerializer.java:77)
	at com.mongodb.util.JSONSerializers$MapSerializer.serialize(JSONSerializers.java:317)

Se ocaciona al ejecutar la función:

$Datasource.addObject(dashboard, "Dashboard")
      .then(response => {
        $state.go('dashboard.dashboards', {});
});

Donde el objeto dashboard es:

{
"id":1488225282487, 
"name":"dashboard1",
"widgets":{
  "0":{"id":1488225283673,"name":"widget"},
  "1":{"id":1488225284428,"name":"widget"},
  "2":{"id":1488225284590,"name":"widget"}
  }
}

La función que replica el error está ligada al boton save en la ruta:
http://localhost:8080/app/#/dashboard/add/dashboard
En la rama: panelDashboards

**Comment by haxdai** [comment]
@juanCarlos123 el json del objeto dashboard debería ser algo como:

````json
{
  "_id": "1488225282487",
  "name": "dashboard1",
  "widgets": [
    {
      "id": 1488225283673,
     "name":"widget"
    },
    {
      "id": 1488225284428,
     "name":"widget"
    },
    {
      "id": 1488225284590,
     "name":"widget"
    }
  ]
}
````
Recuerda que los IDs los genera el servicio $Datasource al realizar la inserción y son strings.

**Comment by haxdai** [comment]
@juanCarlos123 así puedes referenciar una línea en particular de un código para ir al punto:

https://github.com/FST2015PM/Plataforma/blob/panelDashboards/src/main/webapp/app/js/controllers/dashboards/editdashboard.controller.js#L133

**Comment by juanCarlos123** [comment]
Perfecto. Cambié el JSON de la siguiente manera:
{
  "name": "dashboard1",
  "widgets": [
    {
      "id": 1488225283673,
     "name":"widget"
    },
    {
      "id": 1488225284428,
     "name":"widget"
    },
    {
      "id": 1488225284590,
     "name":"widget"
    }
  ]
}

mismo error:

Feb 27, 2017 11:27:25 PM org.apache.catalina.core.StandardWrapperValve invoke
SEVERE: Servlet.service() for servlet [Jersey REST Service] in context with path [] threw exception [java.lang.RuntimeException: json can't serialize type : class org.json.JSONObject] with root cause
java.lang.RuntimeException: json can't serialize type : class org.json.JSONObject
	at com.mongodb.util.ClassMapBasedObjectSerializer.serialize(ClassMapBasedObjectSerializer.java:77)
	at com.mongodb.util.JSONSerializers$IterableSerializer.serialize(JSONSerializers.java:290)
	at com.mongodb.util.ClassMapBasedObjectSerializer.serialize(ClassMapBasedObjectSerializer.java:79)
	at com.mongodb.util.JSONSerializers$MapSerializer.serialize(JSONSerializers.java:317)
	at com.mongodb.util.ClassMapBasedObjectSerializer.serialize(ClassMapBasedObjectSerializer.java:79)
	at com.mongodb.util.JSONSerializers$MapSerializer.serialize(JSONSerializers.java:317)
	at com.mongodb.util.ClassMapBasedObjectSerializer.serialize(ClassMapBasedObjectSerializer.java:79)
	at com.mongodb.util.JSON.serialize(JSON.java:54)

**Comment by haxdai** [comment]
Fixed by 7d9577eb936606230f163bc3a1b9a1a7d7137be9.
